vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -134,15 +135,30 @@ void LedgerImpl::Initialize(
 
   initializing_ = true;
 
-  InitializeConfirmations(execute_create_script, callback);
+  confirmations::_environment = ledger::_environment;
```

Comment:
nit: can we move this to `MaybeInitializeConfirmations` so that it's easier to maintain?

Reply:
Moved